Add setting recommendation for KNN getting started guide (#9912)
opensearch-project/documentation-website: _vector-search/getting-started/index.md

```
{% include copy-curl.html %}

> Vector queries usually have a `size` > 0, so by default they don't enter the request cache. In OpenSearch 2.19 or later, if your workload mostly consists of vector queries, consider increasing the dynamic `indices.requests.cache.maximum_cacheable_size` cluster setting  to a larger value, such as `256`. This allows queries with a `size` up to 256 to enter the request cache, improving performance. For more information, see [Request cache]({{site.url}}{{site.baseurl}}/search-plugins/caching/request-cache).

Reviewer: ```suggestion
Vector queries usually have a `size` > 0, so by default they don't enter the request cache. In OpenSearch 2.19 or later, if your workload mostly consists of vector queries, consider increasing the dynamic `indices.requests.cache.maximum_cacheable_size` cluster setting to a larger value, such as `256`. This allows queries with a `size` of up to 256 to enter the request cache, improving performance. For more information, see [Request cache]({{site.url}}{{site.baseurl}}/search-plugins/caching/request-cache).
```

> Vector queries usually have a `size` > 0, so by default they don't enter the request cache. If you're using OpenSearch 2.19 or later and your workload will be mostly vector queries, you may want to set the dynamic cluster setting `indices.requests.cache.maximum_cacheable_size` to a larger value like `256`. This allows any queries with `size` up to 256 to enter the request cache, improving performance. For more information, see [related page]({{site.url}}{{site.baseurl}}/search-plugins/caching/request-cache).

Reviewer 2: ```suggestion
Vector queries usually have a `size` > 0, so by default they don't enter the request cache. In OpenSearch 2.19 or later, if your workload mostly consists of vector queries, consider increasing the dynamic `indices.requests.cache.maximum_cacheable_size` cluster setting  to a larger value, such as `256`. This allows queries with a `size` up to 256 to enter the request cache, improving performance. For more information, see [Request cache]({{site.url}}{{site.baseurl}}/search-plugins/caching/request-cache).
```
